source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -354,12 +354,14 @@ locally-managed master key.
 C. Define a JSON Schema
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-`JSON Schema <http://json-schema.org/>`_ is a vocabulary that allows you
-to annotate and validate JSON documents. MongoDB drivers use an extended
-version of the JSON Schema standard to configure automatic client-side
-encryption and decryption of specific fields of the documents in a
-collection. The extended MongoDB CSFLE standard requires the following
-information:
+In this section, MedcoMD engineers configure the fields that the client
+automatically encrypts and decrypts using JSON Schema.
+`JSON Schema <http://json-schema.org/>`_, a vocabulary
```

Comment:
I feel like it reads a little awkwardly to have JSON Schema stated twice back-to-back. Would it be better to phrase it like this (?) :

In this section, MedcoMD engineers configure the fields that the client automatically encrypts and decrypts using `JSON Schema <http://json-schema.org/>`_, a vocabulary that allows you to annotate and validate JSON documents.

Reply:
That was my original edit, but decided the sentence contained too much information (both what is being accomplished by defining a JSON Schema and what JSON Schema is) which resulted in me creating a grammatical error in my manual undo which Nick spotted.

Reply:
That said, if you think it's of value to join in a single sentence, I will update it.

Reply:
With the revised second sentence this should be good now.